cpp/include/raft/detail/layout_padded_general.hpp

```diff
@@ -29,18 +29,16 @@ namespace experimental {
 enum class StorageOrderType { column_major_t, row_major_t };
 
 template <class Extents>
-MDSPAN_INLINE_FUNCTION constexpr auto padded_row_major_strides(size_t ByteAlignment,
-                                                               size_t valueTypeSize,
+MDSPAN_INLINE_FUNCTION constexpr auto padded_row_major_strides(size_t Alignment,
```

Comment:
Just a neatpick: could you check that raft's clang-tidy does not complain about the naming conventions for the arguments? I believe, the `Alignment` should be lower-case.

(Usually I set up vscode to check this: `clang-tidy -p ${fileWorkspaceFolder}/cpp/build ${file} -extra-arg=-std=c++17`)

Reply:
I have adjusted the specific parameter you mentioned. Running clang-tidy gives me several more warnings like ` warning: invalid case style for constexpr variable 'storage_order' [readability-identifier-naming]`, but the suggested naming `kStorageOrder ` does seemingly not match our conventions looking at other places in the code. Is my call missing the correct `.clang-tidy` options?

Reply:
That's correct `.clang-tidy` and I don't know what's the better option :) I usually choose to blindly follow clang-tidy suggestion, but I wouldn't ask you to do the same